1.0/docs/devguide/tests.md

```diff
@@ -0,0 +1,367 @@
+---
+layout: default
+type: guide
+shortname: Docs
+title: Unit tests
+subtitle: Developer guide
+---
+
+<link rel="import" href="../../../0.5/components/google-youtube/google-youtube.html">
+
+{% include toc.html %}
+
+Use Web Component Tester (`wct`) to unit test your Polymer elements. 
+
+`wct` is an end-to-end testing environment built by the Polymer core team. It
+enables you to test your elements locally, against all of your installed
+browsers, or remotely, via Sauce Labs. `wct` is composed of the following
+tools:
```

Comment:
"is composed of" suggests that these tools comprise the entirety of WCT (and by extension, that all those tools are built by the core team).

I'm having a little trouble summing up, because what exactly WCT is is kind of vague. But I think it's something like this:

Web Component Tester provides a command line interface for running tests (`wct`) and a set of utility functions to make it easier to write tests for web components using a set of popular third-party testing tools:

(Might want to distinguish between Web Component Tester, the package (WCT) and `wct` the command-line utility.)